src/ui/utils/tokenManager.tsx

```diff
@@ -172,37 +172,49 @@ class TokenManager {
       return;
     }
 
-    if (this.auth0Client.isAuthenticated) {
-      if (this.isTokenRequested) {
-        if (refresh) {
-          this.reset();
-        } else {
-          return;
-        }
-      }
-
-      this.isTokenRequested = true;
-      const deferredState = this.deferredState;
+    const wasAuthenticated = this.auth0Client.isAuthenticated;
 
-      const item = window.localStorage.getItem("__cypress");
-      if (item) {
-        const token = JSON.parse(item).body.access_token;
-        this.setState({ token }, deferredState);
+    if (this.isTokenRequested) {
+      if (refresh) {
+        this.reset();
+      } else {
         return;
       }
+    }
 
-      try {
-        const token = await this.fetchToken(refresh);
+    this.isTokenRequested = true;
+    const deferredState = this.deferredState;
 
-        this.setState({ token }, deferredState);
-        if (deferredState === this.deferredState) {
-          this.setupTokenRefresh(token);
-        }
-      } catch (e) {
+    const item = window.localStorage.getItem("__cypress");
```

Comment:
what's cypress doing in there?

Reply:
I dunno. I assume it's legacy automated test workaround code. I'm pretty sure it can be removed. I almost did here actually but didn't want to mix it in with this change.